### ISSUE TITLE
Always asks for token

### ISSUE THREAD

**Issue by muuvmuuv** [harshjv/github-repo-size#36]
Every time I restart my Browser (on latest macOS, using Brave with Chromium), this asks me for the GitHub token.

It would be good to have an options page rather than a prompt, which is terrible UX.

**Comment by AminoffZ** [comment]
I agree, [on my project](https://github.com/AminoffZ/github-repo-size) I have made the user experience a lot nicer :).
![](https://raw.githubusercontent.com/AminoffZ/github-repo-size/main/assets/github-repo-size-popup.png)
